src/__tests__/integration.test.js

```diff
@@ -45,14 +45,14 @@ describe('Filter tests', () => {
 describe('Pricing tests', () => {
   it('will render zero when unchecked', () => {
     const app = mount(<App />)
-    expect(app.find('#total-price').text()).toBe('0')
+    expect(app.find('#total-price').text()).toBe('Total: $0')
```

Comment:
I need to write more tests.  I have coverage for 3 of the files.  I will continue to write more this week.